### DISCUSSION TITLE
AMD Error - Device type privateuseone is not supported for torch.Generator() api.

### DISCUSSION THREAD

**Discussion by Atlas217** [lllyasviel/Fooocus#1084] [General]
need help im a complete newbie but I've followed all the step, or did I miss any?
![Screenshot 2023-12-01 011522](https://github.com/lllyasviel/Fooocus/assets/72188545/af94f887-92b0-43b4-8a9b-596a6ff68281)

**Comment by swapnawebdev** [comment]
getting the same issue now for AMD radeon RX 580 GPU

**Comment by novafluxx** [comment]
Also getting the same with RX 7900 XTX
`RuntimeError: Device type privateuseone is not supported for torch.Generator() api.`

**Comment by Adam-Kosicki** [comment]
I'm having the same problem on my 6950 XT

**Comment by kbrot** [comment]
I solved it with this: https://github.com/lllyasviel/Fooocus/issues/624#issuecomment-1757327660
